tests/test_transforms/test_decomposition/test_model_based.py

```diff
@@ -337,3 +337,32 @@ def test_stride_transform(forward_stride_datasets, decompose_model, context_size
 
     assert not transformed.df.iloc[context_size:10].isna().any().any()
     assert transformed.df.iloc[10:].isna().all().any()
+
+
+@pytest.mark.parametrize("features_to_ignore", (["target"], ["regressor_1"], ["target", "regressor_1"]))
+def test_repetitive_call_before_pipeline_with_ignore(outliers_solid_tsds, features_to_ignore):
```

Comment:
Can you explain why do we need such test?

Reply:
To cover [this](https://github.com/etna-team/etna/actions/runs/10633068062/job/29477322573#step:6:5595) case

Reply:
Can you explain how it arises here?